python/ray/serve/_private/application_state.py

```diff
@@ -38,25 +38,33 @@ def __init__(
             deploy_obj_ref: Task ObjRef of deploying application.
             deployment_time: Deployment timestamp
         """
+
+        self._name = name
+        self._deploy_obj_ref = deploy_obj_ref
+        self._app_msg = ""
+        self._deployment_state_manager = deployment_state_manager
+        self._deployment_params: List[Dict] = []
+        # This set tracks old deployments that are being deleted
+        self._deployments_to_delete = set()
+        self.ready_to_be_deleted = False
+        self.route_prefix = None
+        self.docs_path = None
```

Comment:
are these ones intended to be accessed by external callers because they have no `_` prefix?

if so, better to make the explicitly `@property`s

Reply:
Done